packages/toolkit/src/components/ClonePipelineDialog.tsx

```diff
@@ -86,6 +88,25 @@ export const ClonePipelineDialog = ({
     accessToken,
   });
 
+  const organizations = useUserMemberships({
+    enabled: enabledQuery && me.isSuccess,
+    userID: me.isSuccess ? me.data.id : null,
+    accessToken,
+  });
+
+  const organizationsAndUserList = React.useMemo(() => {
+    const orgsAndUserList = [];
+    if (organizations.isSuccess && organizations.data) {
+      organizations.data.map((org) => {
+        orgsAndUserList.push(org.organization);
+      });
```

Comment:
@iamnamananand996 Map is to use to create another list. If you are not going to use the list it creates. I will suggest using simple forEach or for...loop here